Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYApplePayPaymentProvider.m

```diff
@@ -151,20 +154,21 @@ - (BOOL)isAvailable
 	// checks if the client is setup to use Apple Pay
 	// checks if device hardware is capable of using Apple Pay
 	// checks if the device has a payment card setup
+	
+	Class PaymentClass = [self applePayController];
```

Comment:
Passing around class objects is a bit awkward.

What if we keep a reference to the appropriate class as an ivar, and then provide `-canMakePayments` and `-canMakePaymentsUsingNetworks:`, as (instance) cover methods that call through to framework class?